plugins/filter_aws/aws.c

```diff
@@ -102,6 +102,10 @@ static void expose_aws_meta(struct flb_filter_aws *ctx)
                     "aws." FLB_FILTER_AWS_HOSTNAME_KEY,
                     ctx->hostname);
     }
+
+    // REVIEW: What do we use the flb_env_set for?
+    //  Is it used to 'indirectly' access Hostname in another part of the codebase?
+    //  Do we need to set environmental variables for the EC2 instance tags too?
```

Comment:
So first of all, the comment style here does not match the contributing guidelines: https://github.com/fluent/fluent-bit/blob/master/CONTRIBUTING.md

To the question, flb_env_set allows other plugins or code in Fluent Bit to access the values. Its like a local env for Fluent Bit inside the code. 

Right now, no other code is using the tags, since it doesn't exist. But if you want to take the time to add it, that would allow usage in the future. 

A good format for tag keys might be:

```
aws.ec2.tags.{AWS tag key} 
```

Reply:
Thanks for the clarification.

I've left myself a TODO: https://github.com/fluent/fluent-bit/pull/6065/commits/ee3067faaf955e44bf2c29e81c8274d8bbe44a5a

Reply:
Conversation resolved due to lack of new activity. (Feel free to bring it back.)